llvm/test/MC/AMDGPU/gfx11_asm_vop3_from_vop1.s

```diff
@@ -1,4 +1,4 @@
-// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -mattr=+wavefrontsize32 -show-encoding %s | FileCheck --check-prefix=GFX11 %s
+// RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -mattr=+wavefrontsize32,+real-true16 -show-encoding %s | FileCheck --check-prefix=GFX11 %s
 
```

Comment:
Should there be a test file for fake16 gfx11_asm_vop3_from_vop1-fake16.s ?

Reply:
opened two PR for adding missing test files
gfx11 https://github.com/llvm/llvm-project/pull/106089
gfx12 https://github.com/llvm/llvm-project/pull/106093